types/types.go

```diff
@@ -198,6 +198,7 @@ type Info struct {
 	Images             int
 	Driver             string
 	DriverStatus       [][2]string
+	CustomStatus       [][2]string
```

Comment:
it makes sense, can we call it `CurrentStatus` or `SystemStatus`? I feel like `Custom` is very broad.